src/generic/numberparsing.h

```diff
@@ -209,15 +209,12 @@ static bool parse_float_strtod(const char *ptr, double *outDouble) {
   return true;
 }
 
-really_inline bool is_integer(char c) {
+really_inline constexpr bool is_integer(char c) {
   return (c >= '0' && c <= '9');
   // this gets compiled to (uint8_t)(c - '0') <= 9 on all decent compilers
 }
-
-// We need to check that the character following a zero is valid. This is
-// probably frequent and it is harder than it looks. We are building all of this
-// just to differentiate between 0x1 (invalid), 0,1 (valid) 0e1 (valid)...
-const bool structural_or_whitespace_or_exponent_or_decimal_negated[256] = {
+namespace {
```

Comment:
Oh man, I forgot about this. I bet we could use this instead of namespace internal in a lot of places! I wonder how it shows in doxygen when you do that.